src/app/models/user.model.js

```diff
@@ -2,7 +2,7 @@ import mongoose from 'mongoose';
 import bcrypt from 'bcrypt-nodejs';
 
 let userSchema = mongoose.Schema({
-  username: { type: String, unique: true, validate: /^[a-zA-Z][a-zA-Z0-9-_\.]{3,20}$/},
+  username: { type: String, unique: true, validate: /^[a-zA-Z][a-zA-Z0-9-_.]{3,20}$/ },
```

Comment:
Not in this case because it's inside of a character class (`[]`) it makes no difference. It was a linting error so I had to change it.